examples/01_icetray/01_convert_i3_files.py

```diff
@@ -90,7 +99,7 @@ def main_icecube_upgrade(backend: str) -> None:
             "detector", choices=["icecube-86", "icecube-upgrade"]
         )
 
-        args = parser.parse_args()
+        args, unknown = parser.parse_known_args()
```

Comment:
Any reason why you've changed to use `parse_known_args` instead of `parse_args`?